### PR TITLE
Made Child-therapy page

### PR DESCRIPTION
**Pull request by Vanshiii203**
issue #406 

https://user-images.githubusercontent.com/94471189/192101478-1203da6d-21d8-4a18-880c-0bb2e565869d.mp4

### CODE REVIEW

File: html/childTherapy.html [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+      if (n < 1) {slideIndex = slides.length}
+      for (i = 0; i < slides.length; i++) {
+        slides[i].style.display = "none";
+      }
+      for (i = 0; i < dots.length; i++) {
+        dots[i].className = dots[i].className.replace(" active", "");
+      }
+      slides[slideIndex-1].style.display = "block";
+      dots[slideIndex-1].className += " active";
+    }
```
Comment (Susmita-Dey) [inline html/childTherapy.html:307]: @Vanshiii203 Instead of writing javascript here, can you make a separate file for this?

**Review by Susmita-Dey — CHANGES_REQUESTED** [review]
Create a new file for javascript and link it to the `childTherapy.html`

**Comment by Vanshiii203** [comment]
@Susmita-Dey  I have Made the changes.

**Review by Susmita-Dey — APPROVED** [review]
Awesome!! Great job 😍

**Pull request merged**